excutecommand1.js

```diff
@@ -0,0 +1,4 @@
+function generateNumbers() {
+    console.log(generateNumbers);
+}
+    
```

Comment:
this function should want to move into the exeuteCommand.js file. this file is not want.

Reply:
@nethmiprathibha Delete this file. because this changes already have in the executeCommand.js file